conf/flight_plans/ENAC/fish_outdoor.xml

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE flight_plan SYSTEM "../flight_plan.dtd">
+
+<flight_plan alt="190" ground_alt="185" lat0="43.462976" lon0="1.2733345" max_dist_from_home="50" name="Demo fish nav Muret outdoor" security_height="0.3">
+  <header>
+#include "autopilot.h"
+#include "subsystems/datalink/datalink.h"
```

Comment:
Just curiosity, why is this necessary here? Because of the exceptions in the flightplan?

Reply:
yes, for `datalink_time` variable